src/pages/lett-vanskelig/LettEllerVanskeligSpm.test.js

```diff
@@ -1,12 +1,9 @@
 
 import React from 'react';
-import {shallow} from 'enzyme';
+import { mount, shallow, render } from 'enzyme';
 
 import LettEllerVanskeligSpm from './LettEllerVanskeligSpm';
 
-test('has correct license nr', () => {
-
-    const spm = shallow( <LettEllerVanskeligSpm nextPage={ () => true } /> );
-    expect('AB12345').toBe("AB12345");
+it('renders without crashing', () => {
+    shallow(<LettEllerVanskeligSpm nextPage={ jest.fn() } />);
```

Comment:
Trenger vi enzyme her ?
Holder det ikke bare å kjøre: 

```
  const div = document.createElement('div');
  ReactDOM.render(<LettEllerVanskeligSpm  nextPage = {jest.fn()}/>, div);
  ReactDOM.unmountComponentAtNode(div);
```

Reply:
Begge deler funker, enzyme her gjør vi får en oneliner i stedet for tre linjer.

Reply:
BTW: Enzyme varianten er det som var brukt i CRA-dokumentasjonen.

Reply:
OK, tenkte det ikke var noen vits å dra inn mer enn nødvendig i første omgang, hvis vi faktisk skulle ha brukt noe av den mer avanserte som enzyme gir så hadde jeg skjønt det :)